README.md

```diff
@@ -1,12 +1,18 @@
 # Numenta research papers code and data
 This repository contains reproducible code for selected Numenta papers. It is currently under construction and will eventually include the source code for all the scripts used in Numenta's papers.
 
+### Avoiding Catastrophe: Active Dendrites Enable Multi-task Learning in Dynamic Environments
```

Comment:
Missing link to paper. It should be:
```
### [Avoiding Catastrophe: Active Dendrites Enable Multi-task Learning in Dynamic Environments][10]
```

Reply:
There is no published paper yet

Reply:
You should only merge once is published and the title is linked to the paper. Some people may come to this repo looking for papers before they go to arxiv.